packages/twenty-front/src/modules/object-record/graphql/optimistic-effect-definition/getRecordOptimisticEffectDefinition.ts

```diff
@@ -42,6 +50,7 @@ export const getRecordOptimisticEffectDefinition = ({
           const existingRecord = draft.edges.find(
```

Comment:
this looks wrong:
1) newData (createdRecords) should be an array
2) If it's createdRecords, we shouldn't be trying to find existing record

Reply:
For createdRecords, let's keep it as a safeguard